integreat_cms/static/src/js/pois/poi-actions.ts

```diff
@@ -0,0 +1,68 @@
+import { getCsrfToken } from "../utils/csrf-token";
+
+let initialLongitude: string, initialLatitude: string;
+
+window.addEventListener("load", () => {
+    document.getElementById("id_address")?.addEventListener("keyup", fetchCoordinates);
+    document.getElementById("id_postcode")?.addEventListener("keyup", fetchCoordinates);
+    document.getElementById("id_city")?.addEventListener("keyup", fetchCoordinates);
+
+    document.getElementById("auto-fill-coordinates").addEventListener("input", fetchCoordinates);
+
+    initialLongitude = ( document.getElementById("id_longitude") as HTMLInputElement).value;
+    initialLatitude = (document.getElementById("id_latitude") as HTMLInputElement).value;
```

Comment:
@david-venhoff Thank you for your correction 🙈 so much of typos...

When it's a new poi, blank value will be restored if the box is not checked or unchecked. If the POI is saved with blank coordinates, those will be automatically derived as before.

Reply:
There is a small edge case that if the user creates a new poi, fills in the coordinates manually first and then enters the poi address, then the coordinates would get erased. But since this is probably not going to happen I think the current behavior is fine :)

Reply:
Ah ok, I got it!
I'll fix it too :) Thank you for detecting it.